substratest/client.py

```diff
@@ -80,8 +80,8 @@ def add_compute_plan(self, spec, *args, **kwargs):
     def update_compute_plan(self, spec, *args, **kwargs):
         spec_dict = spec.dict()
         # Remove extra field from data
-        spec_dict.pop("compute_plan_id")
-        return self._client.update_compute_plan(spec.compute_plan_id, spec_dict, *args, **kwargs)
+        spec_dict.pop("key")
+        return self._client.update_compute_plan(spec.key, spec_dict, *args, **kwargs)
```

Comment:
Why did you rename it as `key` and not `compute_plan_key` ? 
This modification occurs multiple time in this PR :)

Reply:
Comment addressed in https://github.com/SubstraFoundation/substra/pull/241#issuecomment-721759361